resources/nl/cemu.po

```diff
@@ -515,182 +626,245 @@ msgstr ""
 msgstr ""
 "Hersteld de laatst bekende venster positie en grootte wanneer Cemu wordt "
 "gestart"
 
-#: GeneralSettings2.cpp:97
+#: GeneralSettings2.cpp
 msgid "Remember pad window position"
 msgstr "Onthoud gamepad venster positie"
 
-#: GeneralSettings2.cpp:99
+#: GeneralSettings2.cpp
 msgid "Restores the last known pad window position and size when opening it"
 msgstr ""
 "Hersteld de laatst bekende venster positie en grootte wanneer Cemu wordt "
 "gestart"
 
-#: GeneralSettings2.cpp:103
+#: GeneralSettings2.cpp
 msgid "Discord Presence"
 msgstr "Discord Presence"
 
-#: GeneralSettings2.cpp:105
+#: GeneralSettings2.cpp
 msgid ""
 "Enables the Discord Rich Presence feature\n"
 "You will also need to enable it in the Discord settings itself!"
 msgstr ""
 "Schakelt de Discord Rich Presence functie in\n"
 "Je zult dit ook in de Discord instellingen aanhebben!"
 
-#: GeneralSettings2.cpp:110
+#: GeneralSettings2.cpp
 msgid "Fullscreen menu bar"
 msgstr "Volledig scherm menu bar"
 
-#: GeneralSettings2.cpp:112
+#: GeneralSettings2.cpp
 msgid ""
 "Displays the menu bar when Cemu is running in fullscreen mode and the mouse "
 "cursor is moved to the top"
 msgstr ""
 "Laat de menu bar zien wanneer Cemu in volledig scherm modus zit en de muis "
 "cursor aan de bovenkant is."
 
-#: GeneralSettings2.cpp:115
+#: GeneralSettings2.cpp GettingStartedDialog.cpp
 msgid "Automatically check for updates"
 msgstr "Controleer automatisch for nieuwe updates"
 
-#: GeneralSettings2.cpp:117
+#: GeneralSettings2.cpp
 msgid "Automatically checks for new cemu versions on startup"
 msgstr "Controleer voor nieuwe Cemu versies tijdens het opstarten van Cemu"
 
-#: GeneralSettings2.cpp:122
+#: GeneralSettings2.cpp
 msgid "Save screenshot"
 msgstr "Sla schermopname op"
 
-#: GeneralSettings2.cpp:124
-msgid "Saves a screenshot directly to a file"
-msgstr "Sla schermopname direct op naar een bestand"
+#: GeneralSettings2.cpp
+msgid ""
+"Pressing the screenshot key (F12) will save a screenshot directly to the "
+"screenshots folder"
+msgstr ""
+"Drukken op de schermopname toets (F12) zal een schermopname opslaan in de "
+"\"screenshots\" map"
 
-#: GeneralSettings2.cpp:134
+#: GeneralSettings2.cpp
 msgid "MLC Path"
 msgstr "MLC pad"
 
-#: GeneralSettings2.cpp:140
+#: GeneralSettings2.cpp
 msgid ""
 "The mlc directory contains your save games and installed game update/dlc data"
 msgstr ""
 "Het MLC pad bevat je save games en de door jouw geïnstalleerde game updates/"
 "DLC's"
 
-#: GeneralSettings2.cpp:146
+#: GeneralSettings2.cpp
 msgid ""
 "Select a custom mlc path\n"
 "The mlc path is used to store Wii U related files like save games, game "
 "updates and dlc data"
 msgstr ""
-"Selecteer een aangepaste mlc pad\n"
-"De mlc pad wordt gebruikt om Wii U gerelateerde bestanden zoals save games, "
+"Selecteer een aangepaste mlc map\n"
+"De mlc map wordt gebruikt om Wii U gerelateerde bestanden zoals save games, "
 "game updates en dlc data."
```

Comment:
'mlc' should be uppercase here for consistency with the other translated strings